src/components/DashboardHeader.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import DashboardCard from './DashboardCard';
+
+import rejectedCard from '../Images/Rejected Box.png';
+import scheduledCard from '../Images/Scheduled Box.png';
+import acceptedCard from '../Images/Accepted Box.png';
```

Comment:
For file directory consistency Images directory should be images. Also try to avoid spaces in file names because they can get a little weird to parse on UNIX machines.

Are these the real assets too btw - they look a little blurry?

Reply:
These are the assets I grabbed from Figma. I fixed the scaling but I'm sure that better images will be created in the future that we can add.